lib/dal/src/workspace_snapshot/graph/detect_updates.rs

```diff
@@ -13,7 +13,7 @@ use crate::{
     EdgeWeight, EdgeWeightKind, EdgeWeightKindDiscriminants,
 };
 
-use super::WorkspaceSnapshotGraphV2;
+use super::WorkspaceSnapshotGraphV3;
```

Comment:
Maybe we want the type aliased name used everywhere (i.e. `WorkspaceSnapshotGraphVCurrent`)?

Reply:
I kind of left this one since I'm not sure whether detect updates may end up living as a thing that's part of the version of the graph format. (So `workspace_snapshot/graph/v57/detect_updates.rs`.) The idea was that anything written against a specific version of the graph would not use the `VLatest` type alias.

Ideally, that `VLatest` alias wouldn't be necessary at all, but we have a ton of code living in places outside of `WorkspaceSnapshot` & `WorkspaceSnapshotGraph` that have intimate knowledge of the internal workings of `WorkspaceSnapshotGraph`. I'd like to keep us moving in a direction of pushing all of that code down into the versioned `WorkspaceSnapshotGraphV...` with higher-level interfaces exposed out through `WorkspaceSnapshot`. So, basically the idea we started with at the very beginning of the graph work, but that we (as a group) weren't great on keeping up on.

Reply:
Gotcha, that sounds good. I can see why the alias is a good stopgap for now then.